packages/ui/lib/components/SmoothImage/index.tsx

```diff
@@ -1,10 +1,17 @@
 import React, { useCallback, useState } from 'react';
-import PropTypes from 'prop-types';
 
-import artPlaceholder from '../../../resources/media/art_placeholder.png';
 import styles from './styles.scss';
 
-const SmoothImage = ({
+type SmoothImageProps = {
+  src: string;
+  placeholder?: string;
+  alt?: string;
+}
+
+const artPlaceholder = '../../../resources/media/art_placeholder.png';
```

Comment:
It won't use the correct image like this - it needs to be imported for webpack to load it.

Reply:
When we import it takes the type of `NodeModule` and src of img tag expects a `string` type. If we give src a string it will be able to import the image directly.